client/my-sites/site-settings/composing/markdown.jsx

```diff
@@ -21,10 +21,11 @@ const Markdown = ( { fields, handleToggle, isRequestingSettings, isSavingSetting
 					'Use Markdown syntax to compose content with links, lists, and other styles. This setting enables Markdown in the Classic Editor as well as within a Classic Editor block.'
 				) }
 				link={
-					isJetpack && ! isAtomic
+					siteIsJetpack && ! isAtomic
```

Comment:
I'm not sure about this condition. The markdown info has an incorrect privacy link on simple sites.